desktop-app/app/reducers/browser.js

```diff
@@ -332,6 +335,7 @@ export default function browser(
     windowSize: getWindowSize(),
     allDevicesMuted: false,
     networkConfiguration: _getNetworkConfiguration(),
+    allDevicesInDesignMode: false,
```

Comment:
I think we should also set individual device status to `false` at end of `_getActiveDevices` function.

Reply:
I think it will be _falsy_ by default, but yes is better to set it `false`